app/controllers/spotlight/catalog_controller.rb

```diff
@@ -51,7 +51,10 @@ def autocomplete
   def admin
     self.blacklight_config.view.select! { |k,v| k == :admin_table }
     self.blacklight_config.view.admin_table.partials = [:index_compact]
-    self.blacklight_config.sort_fields.reject! { |k,v| true }
+
+    unless self.blacklight_config.sort_fields.has_key? :timestamp
+      self.blacklight_config.add_sort_field :timestamp, sort: "#{blacklight_config.index.timestamp_field} desc"
```

Comment:
We can't have this sort field configured only on catalog#admin because if you click on an item from a timestamp sorted result set the Previous/Next pagination on the record page will attempt to sort via a config that doesn't exist (which will default to the `timestamp` key, which will fail because it doesn't exist in solr).